### DISCUSSION TITLE
Keymap conflict between Snacks/Scratch and Dadbod

### DISCUSSION THREAD

**Discussion by pogopaule** [LazyVim/LazyVim#5232] [Q&A]
`<leader>S` is used by both Snacks/Scratch (select scratch buffer) and Dadbod (execute query).

In my view that should be resolved. Agree? If so, any idea how?

**Comment by dpetka2001** [comment]
I see that this mapping comes from `vim-dadbod-ui`. According to its [docs](https://github.com/kristijanhusak/vim-dadbod-ui?tab=readme-ov-file#mappings) you can override default mappings in an autocmd. So, probably that is the best way? Something like 
```lua
vim.api.nvim_create_autocmd("FileType", {
    pattern = "dbui",
	group = vim.api.nvim_create_augroup("vim_dadbod_ui", {}),
	desc = "Override dadbod_ui mappings",
	callback = function()
		vim.keymap.set({ "n", "v" }, "<localleader>S", "<Plug>(DBUI_ExecuteQuery)", { buffer = true })
	end,
})
```
I believe should do it? I don't do SQL myself, so you might want to try it out to see if it actually works.
